test/runtests.jl

```diff
@@ -26,7 +26,7 @@ max_worker_rss != typemax(Csize_t) && move_to_node1("parallel")
 cd(dirname(@__FILE__)) do
     n = 1
     if net_on
-        n = min(8, Sys.CPU_CORES, length(tests))
```

Comment:
Probably sth related to julia.mit.edu? It's before `JULIA_CPU_CORES` is added so I suppose any similar needs can be addressed with the env now.